src/angular/src/app/spark-core-angular/components/sprk-tabbed-navigation/sprk-tabbed-navigation.component.ts

```diff
@@ -0,0 +1,114 @@
+import { Component, Input, ContentChildren, QueryList, AfterContentInit, HostListener, ElementRef } from '@angular/core';
+import { SprkTabbedNavigationTabDirective } from '../../directives/tabbed-navigation/sprk-tabbed-navigation-tab/sprk-tabbed-navigation-tab.directive';
+import { SprkTabbedNavigationPanelDirective } from '../../directives/tabbed-navigation/sprk-tabbed-navigation-panel/sprk-tabbed-navigation-panel.directive';
+import * as _ from 'lodash';
+import { resetTabs, setActiveTab, retreatTab, advanceTab, getActiveTabIndex, ariaOrientation } from '@sparkdesignsystem/spark-core/components/tabs';
+
+@Component({
+  selector: 'sprk-tabbed-navigation',
+  template: `
+    <div [ngClass]="getClasses()" role="tablist">
+      <div class="sprk-c-Tabs__buttons">
+        <ng-content select="[sprk-tabbed-navigation-tab]"></ng-content> 
```

Comment:
Might want to add ability for analytics input here so we can track clicks on the tab views.